conda/recipes/librmm/meta.yaml

```diff
@@ -18,6 +18,13 @@ build:
     - CUDAHOSTCXX
     - PARALLEL_LEVEL
     - VERSION_SUFFIX
+    - CCACHE_DIR
+    - CCACHE_NOHASHDIR
+    - CCACHE_COMPILERCHECK
+    - CMAKE_GENERATOR
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
```

Comment:
We should really make a rapids-build package or something similar that sets these via an activation script instead of passing all of these through the recipe, but we can do that in follow ups I suppose.

Reply:
I'm not sure I understand what you mean. Are you talking about utilizing this: https://conda.io/projects/conda-build/en/latest/resources/activate-scripts.html

Reply:
Yes exactly, we could have a package with an `activate` script that sets all of these environment variables as expected and then depend on the package in the `build` section or something similar.